Streamline AWS migration overview and update transport links (#7229)
Particular/docs.particular.net: modernization/migrate-to-the-cloud/on-premises-to-aws.md

[AWS Application Migration Service](https://aws.amazon.com/application-migration-service/when-to-choose-aws-mgn/) is an option for getting on-premises systems into the cloud quickly.

Amazon Simple Queue Service ([Amazon SQS](https://aws.amazon.com/sqs/)) is a fully managed cloud-based queueing service supported by the [NServiceBus Amazon SQS Transport](/transports/sqs).

The [NServiceBus Messaging Bridge](/nservicebus/bridge) acts as a connector, enabling seamless and reliable message exchange between on-premises endpoints and those using the Amazon SQS Transport in AWS. This allows you to migrate endpoints to AWS gradually, without disrupting existing operations.

## AWS supported transport

> - [Amazon SQS Transport](/transports/sqs/)

Reviewer: seee https://docs.particular.net/transports/rabbitmq/#broker-compatibility-hosted-broker-options

[AWS Application Migration Service](https://aws.amazon.com/application-migration-service/when-to-choose-aws-mgn/) is an option for getting on-premises systems into the cloud quickly.

> Amazon Simple Queue Service ([Amazon SQS](/transports/sqs)) is a fully managed cloud-based queueing service supported by NServiceBus. This allows teams to gradually migrate workloads from on-premises systems to AWS, reducing risk and downtime. Hybrid scenarios are fully supported, so on-premises and cloud-based endpoints can communicate reliably during migration.

Reviewer 2: The paragraph below already talks about connecting on-premises to cloud at your own pace, so I don't know that we have to mention it here too.  Also, I thought I'd suggest adding a link to the AWS SQS page as well as our own transport.

```suggestion
Amazon Simple Queue Service ([Amazon SQS](https://aws.amazon.com/sqs/)) is a fully managed cloud-based queueing service supported by the [NServiceBus Amazon SQS Transport](/transports/sqs).
```

The [NServiceBus Messaging Bridge](/nservicebus/bridge) acts as a connector, allowing on-premises endpoints to exchange messages seamlessly and reliably with those using the Amazon SQS Transport. This makes it possible to move endpoints to AWS at your own pace.

## AWS supported transport

- [Amazon SQS Transport](/transports/sqs/)

## On premise transports

- [MSMQ](http://localhost:55666/transports/msmq/)
- [RabbitMQ](http://localhost:55666/transports/rabbitmq/)
- [SQL Server](http://localhost:55666/transports/sql/)
- [PostgreSQL](http://localhost:55666/transports/postgresql/)

> - [Learning Transport](http://localhost:55666/transports/learning/)

Reviewer 2: I'd remove the learning transport because we don't recommend using it in a production environment.

```suggestion
```
